_src/static/sass/components/_copy.scss

```diff
@@ -0,0 +1,103 @@
+// ------------------------------
+// edX Pattern Library: Components - Copy
+
+// About: Contains base styling for copy (general text and type used in content and functionaly).
+
+// #GLOBAL
+// #INDIVIDUAL COPY CASES
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+.copy {
+    @extend %reset-copy;
+    color: $text-base-color;
+
+    &.emphasized {
+        color: $text-emphasized-color;
+        font-weight: $text-emphasized-font-weight;
+    }
+
+    &.de-emphasized {
+        color: $text-de-emphasized-color;
+        font-weight: $text-de-emphasized-font-weight;
+    }
+}
+
+// ----------------------------
+// #INDIVIDUAL COPY CASES
+// ----------------------------
+// archetypes
+%copy-lead {
+    @extend %reset-copy;
+    font-size: font-size(large);
+    line-height: line-height(mid-large);
+    color: $text-base-color;
+
+    p, ol, ul, dl {
+        margin-bottom: spacing-vertical(mid-small);
+    }
+}
+
+%copy-large {
+    @extend %reset-copy;
+    font-size: font-size(mid-large);
+    line-height: line-height(mid-large);
+
+    p, ol, ul, dl {
+        margin-bottom: spacing-vertical(small);
+    }
+}
+
+%copy-base {
+    @extend %reset-copy;
+    font-size: font-size(base);
+    line-height: line-height(mid-large);
+
+    p, ol, ul, dl {
+        margin-bottom: spacing-vertical(small);
+    }
+}
+
+%copy-meta {
+    @extend %reset-copy;
+    font-size: font-size(small);
+    line-height: line-height(mid-large);
+
+    p, ol, ul, dl {
+        margin-bottom: spacing-vertical(small);
+    }
+}
+
+%copy-micro {
+    @extend %reset-copy;
+    font-size: font-size(x-small);
+    line-height: line-height(small);
+
+    p, ol, ul, dl {
+        margin-bottom: spacing-vertical(small);
+    }
+}
+
+// application of archetypes
```

Comment:
NOTE: It made sense to create Sass extends above (which could be silently extended when we need them in future/more complicated patterns as well as provide utility classes (that merely use the extends here).